index.src.html

```diff
@@ -670,72 +670,75 @@ <h3 id="settings-object">
     Is |settings object| a secure context?
   </h3>
 
-  Given a <a>settings object</a> (|settings object|), this algorithm returns
+  Given a <a>settings object</a>, |settings object|, this algorithm returns
```

Comment:
My recommendation would be to use settingsObject for the variable name, although I guess you might still need some kind of styling as you won't always have two words.